doc/source/cluster/kubernetes.rst

```diff
@@ -148,7 +148,7 @@ To get started, we need to apply the `Kubernetes Custom Resource Definition`_ (C
 
 .. code-block:: shell
 
- $ kubectl apply -f ray/python/ray/autoscaler/kubernetes/operator_configs/cluster_crd.yaml
+ $ kubectl apply -f https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kubernetes/operator_configs/cluster_crd.yaml
```

Comment:
When I tried it with the full URL, I got 
`error: error parsing https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/kubernetes/operator_configs/cluster_crd.yaml: error converting YAML to JSON: yaml: line 92: mapping values are not allowed in this context`

@vmarkovtsev did the command with the full URL work on your end?

Reply:
I ran into the same error while applying the cluster CRD yaml

Reply:
Just checking -- did you see the error when specifying the CRD by url or when specifying a local file?